MAINTAINERS.md

```diff
@@ -5,18 +5,19 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 ## Current Maintainers
 
 | Maintainer           | GitHub ID                                           | Affiliation |
-|----------------------|-----------------------------------------------------|-------------|
-| Bhavana Goud Ramaram | [rbhavna](https://github.com/rbhavna)               | Amazon      |
-| Charlie Yang         | [model-collapse](https://github.com/model-collapse) | Amazon      |
-| Dhrubo Saha          | [dhrubo-os](https://github.com/dhrubo-os)           | Amazon      |
-| Jing Zhang           | [jngz-es](https://github.com/jngz-es)               | Amazon      |
-| Junshen Wu           | [wujunshen](https://github.com/wujunshen)           | Amazon      |
-| Sicheng Song         | [b4sjoo](https://github.com/b4sjoo)                 | Amazon      |
-| Xun Zhang            | [Zhangxunmt](https://github.com/Zhangxunmt)         | Amazon      |
-| Yaliang Wu           | [ylwu-amzn](https://github.com/ylwu-amzn)           | Amazon      |
-| Zan Niu              | [zane-neo](https://github.com/zane-neo)             | Amazon      |
-| Austin Lee           | [austintlee](https://github.com/austintlee)         | Aryn        |
-| Henry Lindeman       | [HenryL27](https://github.com/HenryL27)             | Aryn        |
+|----------------------|-----------------------------------------------------|-----------|
+| Bhavana Goud Ramaram | [rbhavna](https://github.com/rbhavna)               | Amazon    |
+| Charlie Yang         | [model-collapse](https://github.com/model-collapse) | Amazon    |
+| Dhrubo Saha          | [dhrubo-os](https://github.com/dhrubo-os)           | Amazon    |
+| Jing Zhang           | [jngz-es](https://github.com/jngz-es)               | Amazon    |
+| Junshen Wu           | [wujunshen](https://github.com/wujunshen)           | Amazon    |
+| Sicheng Song         | [b4sjoo](https://github.com/b4sjoo)                 | Amazon    |
+| Xun Zhang            | [Zhangxunmt](https://github.com/Zhangxunmt)         | Amazon    |
+| Yaliang Wu           | [ylwu-amzn](https://github.com/ylwu-amzn)           | Amazon    |
+| Zan Niu              | [zane-neo](https://github.com/zane-neo)             | Amazon    |
+| Austin Lee           | [austintlee](https://github.com/austintlee)         | Aryn      |
+| Henry Lindeman       | [HenryL27](https://github.com/HenryL27)             | Aryn      |
+| Samuel Herman        | [samuel-oci](https://github.com/samuel-oci/)        | Oracle          |
```

Comment:
@samuel-oci help review if these information is correct ?

Reply:
lgtm